inspec.yml

```diff
@@ -6,7 +6,7 @@ copyright_email: you@example.com
 license: Apache-2.0
 summary: "This Security Technical Implementation Guide is published as a tool to improve the security of Department of Defense (DoD) information systems. The requirements are derived from the National Institute of Standards and Technology (NIST) 800-53 and related documents. Comments or proposed revisions to this document should be sent via email to the following address: disa.stig_spt@mail.mil."
 version: 1.3.0
```

Comment:
you need to bump this as well to 1.3.1 so the profile deps pull for external users